### ISSUE TITLE
Tensorflow object detection API training errors

### ISSUE THREAD

**Issue by Munia-AK** [tensorflow/models#6224]
Hello everyone 
following up with TensorFlow Object Detection API Tutorial from sentdex on youtube, I'm trying to run the training command:

**_python train.py --logtostderr --train_dir=training/ --pipeline_config_path=training/ssd_mobilenet_v1_pets.config_**

I'm working on windows 10 and tried running it once with tensorflow-gpu and once with tensorflow cpu but both gave me warnings and errors:

**_WARNING:tensorflow:From C:\Users\HP\AppData\Local\conda\conda\envs\tf-gpu\lib\site-packages\tensorflow\python\platform\app.py:125: main (from __main__) is deprecated and will be removed in a future version.
Instructions for updating:
Use object_detection/model_main.py.
WARNING:tensorflow:From C:\Users\HP\AppData\Local\conda\conda\envs\tf-gpu\lib\site-packages\object_detection-0.1-py3.6.egg\object_detection\legacy\trainer.py:266: create_global_step (from tensorflow.contrib.framework.python.ops.variables) is deprecated and will be removed in a future version.
Instructions for updating:
Please switch to tf.train.create_global_step
WARNING:tensorflow:num_readers has been reduced to 1 to match input file shards.
WARNING:tensorflow:From C:\Users\HP\AppData\Local\conda\conda\envs\tf-gpu\lib\site-packages\object_detection-0.1-py3.6.egg\object_detection\builders\dataset_builder.py:80: parallel_interleave (from tensorflow.contrib.data.python.ops.interleave_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use `tf.data.experimental.parallel_interleave(...)`.
WARNING:tensorflow:From C:\Users\HP\AppData\Local\conda\conda\envs\tf-gpu\lib\site-packages\tensorflow\python\ops\sparse_ops.py:1165: sparse_to_dense (from tensorflow.python.ops.sparse_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Create a `tf.sparse.SparseTensor` and use `tf.sparse.to_dense` instead.
WARNING:tensorflow:From C:\Users\HP\AppData\Local\conda\conda\envs\tf-gpu\lib\site-packages\object_detection-0.1-py3.6.egg\object_detection\core\preprocessor.py:1218: calling squeeze (from tensorflow.python.ops.array_ops) with squeeze_dims is deprecated and will be removed in a future version.
Instructions for updating:
Use the `axis` argument instead
WARNING:tensorflow:From C:\Users\HP\AppData\Local\conda\conda\envs\tf-gpu\lib\site-packages\object_detection-0.1-py3.6.egg\object_detection\core\batcher.py:96: batch (from tensorflow.python.training.input) is deprecated and will be removed in a future version.
Instructions for updating:
Queue-based input pipelines have been replaced by `tf.data`. Use `tf.data.Dataset.batch(batch_size)` (or `padded_batch(...)` if `dynamic_pad=True`).
WARNING:tensorflow:From C:\Users\HP\AppData\Local\conda\conda\envs\tf-gpu\lib\site-packages\tensorflow\python\training\input.py:751: QueueRunner.__init__ (from tensorflow.python.training.queue_runner_impl) is deprecated and will be removed in a future version.
Instructions for updating:
To construct input pipelines, use the `tf.data` module.
WARNING:tensorflow:From C:\Users\HP\AppData\Local\conda\conda\envs\tf-gpu\lib\site-packages\tensorflow\python\training\input.py:751: add_queue_runner (from tensorflow.python.training.queue_runner_impl) is deprecated and will be removed in a future version._**

Also it gives me zero as result:

**_INFO:tensorflow:depth of additional conv before box predictor: 0
INFO:tensorflow:depth of additional conv before box predictor: 0
INFO:tensorflow:depth of additional conv before box predictor: 0
INFO:tensorflow:depth of additional conv before box predictor: 0
INFO:tensorflow:depth of additional conv before box predictor: 0
INFO:tensorflow:depth of additional conv before box predictor: 0_**

could someone please help?

**Comment by rootkitchao** [comment]
It seems that the path format you used in the command is wrong.This should look like this:
python legacy/train.py --logtostderr --num_clones=2 --ps_tasks=1 --train_dir=D:\tf_project\plate_detector\model\train --pipeline_config_path=D:\tf_project\plate_detector\model\ssd_mobilenet_v2_fpn_640.config

**Comment by pkulzc** [comment]
Those are all warnings that you can ignore. Is there an error?

Also train.py has been deprecated for a while. Please try using object_detection/model_main.py if possible.
